src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/ConfirmOfflinePaymentSubmissionViewModelTests.cs

```diff
@@ -0,0 +1,100 @@
+namespace EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.RegistrationSubmissions
+{
+    using EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.Shared.Helpers;
+    using EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+
+    [TestClass]
+    public class ConfirmOfflinePaymentSubmissionViewModelTests
+    {
+        [TestMethod]
+        public void ConfirmOfflinePaymentSubmissionViewModel_ValidModel_ShouldHaveNoValidationErrors()
```

Comment:
Can you use 'DataRow' cover more scenarios in one TestMethod?

Reply:
Done